client/signup/steps/website-content/section-generator.tsx

```diff
@@ -42,6 +47,38 @@ export const sectionGenerator = ( params: SectionGeneratorReturnType< WebsiteCon
 			},
 		};
 	} );
+	return { elapsedSections: elapsedSections + formValues.pages.length, websiteContentSections };
+};
+const generateLogoSection = (
+	params: SectionGeneratorReturnType< WebsiteContent >,
+	elapsedSections = 0
+) => {
+	const { translate, formValues } = params;
+
+	const fieldNumber = elapsedSections + 1;
+	const pageTitle = 'Site logo';
+	return {
+		title: translate( '%(fieldNumber)d. %(pageTitle)s', {
```

Comment:
We'll need to add `Site logo` here in the string to be translated, not as a variable. It won't get translated otherwise. Also, can we change it to the title case - `Site Logo`?

Reply:
Ah ya good catch :)